Samples/word-import-template/README.md

```diff
@@ -48,67 +65,101 @@ The user updates their Word document with the content from another Word document
+### Use GitHub as the web host
 
-### Try it out
+The quickest way to run the sample is to use GitHub as the web host. However, you can't debug or change the source code. The add-in web files are served from this GitHub repository.
 
-Once the add-in is loaded, use the following steps to try out the functionality.
+1. Download the **word-import-template.zip** file from this sample to a folder on your computer.
+1. Sideload the add-in manifest in Word by following the appropriate instructions in the article [Sideload Office Add-ins that use the unified manifest for Microsoft 365](https://learn.microsoft.com/office/dev/add-ins/testing/sideload-add-in-with-unified-manifest).
+1. Follow the steps in [Try it out](#try-it-out) to test the sample.
 
-1. Open Word on Windows, on Mac, or in a browser.
+### Use localhost
 
-1. To open the add-in task pane, go to the **Home** tab and choose **Show Task Pane**.
+If you prefer to host the web server on localhost, follow these steps.
 
-1. In the "Template" section of the add-in UI, select **Choose File**. Navigate to the location of your .docx file then open the file. The template is automatically applied to your document, replacing any preexisting content.
+1. Clone or download this repository.
+1. From a command prompt, go to the root of the project folder **/samples/word-import-template**.
+1. Run the following commands.
 
-    ![The initial screen displaying the button to choose a file.](./resources/word-import-template-initial-screen.png)
+    ```console
+    npm install
+    ```
 
-    For convenience, the resources folder of this project includes a *template example.docx* file.
+    ```console
+    npm start
+    ```
 
-    ![Screen showing the imported template.](./resources/word-import-template-applied.png)
+    This starts the web server on localhost and sideloads the **manifest.json** file to Word.
 
-1. In the document, update the text and other content.
+1. Follow the steps in [Try it out](#try-it-out) to test the sample.
 
-## Run the sample from localhost
+1. To stop the web server and uninstall the add-in from Word, run the following command.
 
-If you prefer to configure a web server and host the add-in's web files from your computer, use the following steps.
+    ```console
+    npm stop
+    ```
 
-1. Install a recent version of [npm](https://www.npmjs.com/get-npm) and [Node.js](https://nodejs.org/) on your computer. To verify if you've already installed these tools, run the commands `node -v` and `npm -v` in your terminal.
+### Use Microsoft Azure
 
-1. You need http-server to run the local web server. If you haven't installed this yet, run the following command.
+You can deploy this sample with the unified manifest to Microsoft Azure using the Teams Toolkit extension in Visual Studio Code.
 
-    ```console
-    npm install --global http-server
-    ```
+1. In Visual Studio Code, go to the activity bar, then open the Teams Toolkit extension.
```

Comment:
Same here and throughout the file. "Teams Toolkit" should be "Microsoft 365 Agents Toolkit"

Reply:
```suggestion
1. In Visual Studio Code, go to the activity bar, then open the Microsoft 365 Agents Toolkit extension.
```